### PR TITLE
Style: updated scroll to top button position

### PR DESCRIPTION
**Pull request by Sen-442b**
positioned  the fab button  towards the bottom right corner of the screen for better  UX  by mobile users
www hacksquad dev_(BlackBerry Z30)

### CODE REVIEW

File: src/styles/fab.css [hunk 1]
```diff
@@ -1,7 +1,8 @@
 .fab-btn{
     position: fixed;
-    bottom: 80px;
-    right: 60px;
+    background:#000000bd;
```
Comment (TheLearneer) [inline src/styles/fab.css:3]: Making this change will mess up the position in larger devices, thus what you can do is use media query here to implement responsive design.
OR
You can try using tailwind classes for the responsive design as well.

File: src/styles/fab.css [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
+    transform: translateY(-0.5rem);
+  }
+  100% {
+    transform: translateY(0rem);
+  }
+}
+
+@media (max-width: 600px) {
+  .fab-btn {
+    bottom: 38px;
```
Comment (TheLearneer) [inline src/styles/fab.css:35]: Both the viewports have same `bottom` and `right` values now

**Review by TheLearneer — CHANGES_REQUESTED** [review]

File: src/styles/fab.css (hunk 2 above)
Comment (Sen-442b) [reply src/styles/fab.css:35]: I forgot to update the previous changes, apologies for the inconvenience

**Review by nevo-david — COMMENTED** [review]
Thank you!

**Review by nevo-david — APPROVED** [review]

**Pull request merged**